apps/dot-voting/app/components/VoteDetails/index.js

```diff
@@ -26,7 +26,7 @@ const VoteDetails = ({ vote, onVote }) => {
   const { voteWeights, votingPower } = useUserVoteStats(vote)
   const tokenContract = tokenAddress && api.external(tokenAddress, tokenAbi)
 
-  const network = useNetwork()
+  //const network = useNetwork()
```

Comment:
Can e delete this line, and stop importing `useNetwork`?

Reply:
Oops, meant to delete that after testing!